### PR TITLE
added tests

### PR DESCRIPTION
**Pull request by spriadka**
## Motivation

Write tests

### CODE REVIEW

**Comment by spriadka** [comment]
@wtrocki Please review this PR

File: integration/vehicle-inspection/test/InMemoryTaskRepositoryTest.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                assert.deepEqual(tasks,seedData);
+            });
+        })
+
+    });
+
+    describe('#createMany',function(){
+        it('Should create tasks correctly on empty task repository',function(){
+            inMemoryTaskRepository = new InMemoryTaskRepository();
+            inMemoryTaskRepository.createMany([new BaseTask(), new BaseTask()]);
```
Comment (paolobueno) [inline integration/vehicle-inspection/test/InMemoryTaskRepositoryTest.ts:66]: could move the common steps for each method to a `beforeEach`

**Comment by paolobueno** [comment]
Nice start, are you going to add tests for the other repositories in the example?
If we keep the interfaces in the wfm repo, I'd like to move some of these tests there and use them from the example, similar to what's being done here: https://github.com/feedhenry-raincatcher/raincatcher-core/blob/master/client/wfm/test/Task/BaseTask.ts#L12 (the BaseTask implementation is using tests from the suite that just targets the original interface)

File: integration/vehicle-inspection/test/InMemoryTaskRepositoryTest.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+describe('InMemoryTaskRepository Tests',function(){
+    let inMemoryTaskRepository: InMemoryTaskRepository;
+
+    describe('#construct',function(){
+          it('Should construct repository without any errors',function(){
+            inMemoryTaskRepository = new InMemoryTaskRepository();
+            assert.ok(inMemoryTaskRepository);
+        });
+        it('Should construct repository with no tasks',function(){
```
Comment (paolobueno) [inline integration/vehicle-inspection/test/InMemoryTaskRepositoryTest.ts:13]: can you use 2 spaces for identation?

**Review by paolobueno — CHANGES_REQUESTED** [review]
A few minor comments, and it's missing tests for the other repositories

**Comment by wtrocki** [comment]
Closing as stale PR. @paolobueno Feel free to integrate this into your own work and fix remaining problems.

**Pull request closed without merging**